sed/config/default.yaml

```diff
@@ -32,6 +40,18 @@ dataframe:
   # binning factor used for the adc coordinate (2^(adc_binning-1))
   adc_binning: 2
 
+  # Flash loader requires the daq parameter and offset
+  daq: fl1user3
+  ubid_offset: 5
+  # The h5 file paths and aliases are defined under channels
+  # PulseId channel(s) are necessary
+  channels:
+      pulseId:
```

Comment:
I think this needs to be nested one level less.

Reply:
Yes that's possible. I will make an issue out of that, as flash loader needs to adapt to this.
And then mpes can also be adapted to make use of the channels header.

Reply:
My point is that the whole thing probably does not work, because pulseId is indented two levels from channels (so it appears to me)...

Reply:
@zainsohail04 I fixed this now, at least tests pass again. Please make sure that you don't break things with your commits in the future, or at least fix them quickly. Here I even told you what to do...

Reply:
Hi Laurenz sorry about that. There was still implementation of flash loader that has been changing a lot. Plus, a lot of config handling also changed.
I can revert my commit completely and later add flash defaults once that branch is merged. What do you think?